kornia/augmentation/container/ops.py

```diff
@@ -223,6 +226,65 @@ def transform(cls, input: Tensor, module: Module, param: ParamItem, extra_args:
             return MaskSequentialOps.transform(input, module=module.op, param=param, extra_args=extra_args)
         return input
 
+    @classmethod
+    def transform_list(
+        cls, input: List[Tensor], module: Module, param: ParamItem, extra_args: Dict[str, Any] = {}
+    ) -> List[Tensor]:
+        """Apply a transformation with respect to the parameters.
+
+        Args:
+            input: list of input tensors.
+            module: any torch Module but only kornia augmentation modules will count
+                to apply transformations.
+            param: the corresponding parameters to the module.
+        """
+        if isinstance(module, (K.GeometricAugmentationBase2D,)):
+            tfm_input = []
```

Comment:
What I meant by factoring was to remove boilerplate code of the next four blocks that are practically the same code and put into a single function. Can be done in a subsequent PR